web/templates/navigation_tour_video_modal.hbs

```diff
@@ -4,6 +4,6 @@
         <source src="{{video_src}}" type="video/mp4"/>
     </video>
     <div id="navigation-tour-video-ended-button-wrapper">
-        <button id="navigation-tour-video-ended-button">{{t "Let's go!"}}</button>
+        <button id="navigation-tour-video-ended-button" class="action-button-primary-brand">{{t "Let's go!"}}</button>
```

Comment:
I feel like this should be using this:

```
+        {{> components/action_button
+          label=(t "Let's go!")
+          id="navigation-tour-video-ended-button"
+          attention="primary"
+          intent="brand"
+        }}
```

But visually it looks good, and I've not audited what CSS we should delete doing that. Perhaps you can @sayamsamal can collaborate on cleaning that up.